lib/mozart_fetcher/envelope.ex

```diff
@@ -12,7 +13,8 @@ defmodule MozartFetcher.Envelope do
 
       {:error, _, _} ->
         ExMetrics.increment("error.envelope.decode")
+        Stump.log(:error, %{message: "Failed to decode Envelope"})
```

Comment:
I think it might be good to also log the `envelope`

Reply:
Done, have labelled it `body` and passed through from the `case` above